packages/runtime/test/dataViews/requestItems/ShareAttributeRequestItemDVO.test.ts

```diff
@@ -223,8 +222,8 @@ describe("ShareAttributeRequestItemDVO", () => {
         const baselineNumberOfAttributes = await syncAndGetBaselineNumberOfAttributes(sRuntimeServices, {
             query: { "content.value.@type": "DisplayName", "shareInfo.peer": rAddress }
         });
+        sEventBus.reset();
```

Comment:
the eventbus should be resetted after every test, why is this required?

Reply:
The syncs also process the accept response of the previous test - and the test that should wait for the processing of the current attribute, would instead run after processing of the old attribute causing an error in the status check

Reply:
you can also set filters when waiting for events